mkdocs/tests/structure/page_tests.py

```diff
@@ -342,6 +342,35 @@ def test_page_title_from_setext_markdown(self):
             expected='Welcome to MkDocs Setext',
         )
 
+    _COMMENT_CONTENT = dedent(
+        '''
+        <!--
+        Comment
+        -->
+
+        # Hello
+        '''
+    )
+
+    def test_page_title_from_markdown_after_comment(self):
+        self._test_extract_title(self._COMMENT_CONTENT, expected='Hello')
+
+    _FRONTMATTER_CONTENT = dedent(
+        '''
+        <!--
+        Comment
+        -->
+        ---
+        title: test
+        ---
```

Comment:
It should also be double-checked whether this is actually treated as a frontmatter or just a bunch of visible Markdown. Because I'd guess the latter.

Reply:
You are right, that doesn't work. I'll remove that test case.